### PR TITLE
Add screenshot and fix makefile

### PR DESCRIPTION
**Pull request by localanu**

### CODE REVIEW

File: src/makefile
Comment (scythx) [inline src/makefile]: You should put it in "LIBS=".
```
LIBS= be stdc++ screensaver
```

File: README.md
Comment (scythx) [inline README.md]: I'm not sure but it would be good if you place the screenshoot below the short description.
Example https://github.com/HaikuArchives/sanity.

File: src/makefile
Comment (raefaldhia) [inline src/makefile]: I think this line is save to delete.

**Comment by localanu** [comment]
Okay,Thanks for review
I updated it :smile:

File: src/makefile
Comment (scythx) [inline src/makefile]: it should be
LIBS= be sdc++ screensaver

without `"` and `-l`

**Comment by localanu** [comment]
Updated it

File: src/makefile
Comment (scythx) [inline src/makefile]: Opps, it's `stdc++` not `sdc++`.

File: src/makefile
Comment (scythx) [inline src/makefile]: I think line 115-117 are save to remove.

**Review by scythx — CHANGES_REQUESTED** [review]

**Comment by scythx** [comment]
Don't forget to submit for review in your task, so i can approve. Maybe i'll aprove the task before this PR got merged or we'll wait for @waddlesplash review because you're already waited 14 hours for review :o.

But you can always come back here to fix something in your PR :D.

File: src/makefile
Comment (humdingerb) [reply src/makefile]: Even better, use $(STDCPPLIBS) so it'll work under gcc2 and gcc5.

File: README.md
Comment (humdingerb) [reply README.md]: And, why is the screenshot title "Pe"?  How about "It's full of stars..." :)

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: README.md
Comment (localanu) [reply README.md]: Sorry for hear that,I forget to change name

**Comment by localanu** [comment]
I change name in readme.md

File: README.md
Comment (localanu) [reply README.md]: I changed

File: src/makefile
Comment (humdingerb) [reply src/makefile]: Also, remove the quotes "".
"screensaver" isn't needed either.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by humdingerb** [comment]
There are more changes to be done, see comments by raefaldhia (and me).

**Comment by localanu** [comment]
Updated it

File: src/makefile
Comment (humdingerb) [inline src/makefile]: As I commented before, please change `stdc++` to `$(STDCPPLIBS)`.

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by localanu** [comment]
I updated it

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks.

**Comment by ghost** [comment]
@localanu The task for this also said you have to write a recipe, yet I've not seen it anywhere at haikuports.

**Pull request merged**
